packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -606,7 +606,10 @@ export class DeltaManager
         // But for view-only connection, we have no such signal, and with no traffic
         // on the wire, we might be always behind.
         // See comment at the end of setupNewSuccessfulConnection()
-        this.logger.debugAssert(this.handler !== undefined || !fetchOpsFromStorage); // can't fetch if no baseline
+        logIfFalse(
```

Comment:
I'd personally swap first two args...

Reply:
can do. the method already exists, but i need to make changes after i consume all the bumps, so i'll do it then